jenkins/config-release/settings-release.xml

```diff
@@ -8,4 +8,26 @@
             <password>${repo.password}</password>
         </server>
     </servers>
+
+    <profiles>
+        <profile>
+            <id>symphony-repo</id>
+            <activation>
+                <activeByDefault>false</activeByDefault>
+            </activation>
+
+            <distributionManagement>
```

Comment:
The `<distributionManagement>` block is already defined at line `209`, but declared as part of `release` profile.
Why don't we move it outside of the `release` profile and make default URLs pointing to ossrh? Doing that we would just have only `<distributionManagement>` block declared at the end of this file.

Reply:
Indeed but for ossrh, we need the `nexus-staging-maven-plugin` and the `<autoReleaseAfterClose>true</autoReleaseAfterClose>` configuration in order to have the artifacts released (and not only staged)

Reply:
@symphony-thibault Did you ever tried with this repository URL
```xml
<repository>
  <id>ossrh</id>
  <url>https://oss.sonatype.org/content/repositories/releases</url>
</repository>
```
instead of `https://oss.sonatype.org/service/local/staging/deploy/maven2/` ?